dashboard/src/main/java/org/openedx/dashboard/presentation/DashboardListFragment.kt

```diff
@@ -515,6 +582,8 @@ private fun CourseItemPreview() {
     }
 }
 
+object MockFragmentManager : FragmentManager()
```

Comment:
not a good idea to mock it  as an object inside Fragment, I think it's better do not to put fragmentManager as an argument into DashboardListView, it better to make a lambda, for example
onPaymentConfirmed () -> Unit

Reply:
addressed.